src/pagination/PageLink.ts

```diff
@@ -0,0 +1,40 @@
+import { v } from '@dojo/widget-core/d';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+
+import * as css from '../styles/pagination.m.css';
+
+export const PageLinkBase = ThemeableMixin(WidgetBase);
+
+export interface PageLinkProperties extends ThemeableProperties {
+	disabled?: boolean;
+	isArrow?: boolean;
+	label?: string;
+	page: number;
+	onPageRequest?: (page: number) => void;
+}
+
+@theme(css)
+class PageLink extends PageLinkBase<PageLinkProperties> {
+	onClick (event: any) {
+		this.properties.onPageRequest && this.properties.onPageRequest(this.properties.page);
```

Comment:
`onPageRequest` is being read twice, so it makes sense to assign it to a single variable and if you're doing that, might as well pull out `page` too. The tiniest bit faster and easier to read.